packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -479,14 +479,24 @@ export class ElectronMainApplication {
     }
 
     protected async getForkOptions(): Promise<ForkOptions> {
+        const env: NodeJS.ProcessEnv = {
+            ...process.env,
+            [ElectronSecurityToken]: JSON.stringify(this.electronSecurityToken),
+        };
+
+        // when the ElectronSecurityToken was present on the parent process already, we need to check for variants of the key
+        if (process.platform === 'win32') {
```

Comment:
Is this a transformation that Windows does automatically - it translates all env variables to upper case?

Reply:
I just tried quickly and if I set a env var in lowercase I still see it lowercase in `process.env`.

The weird behaviour on Windows is that `process.env.TEST` is the same as `process.env.test`:

```batch
> set test=1 && node -p "'TEST' in process.env && 'test' in process.env"
true
```